pkg/config/config.go

```diff
@@ -215,30 +215,6 @@ func (c *Config) BasicAuth() (user, pass string, ok bool) {
 	return user, pass, ok
 }
 
-// TLSCertFiles returns certificate and key files and an error if
-// both files doesn't exist and have approperiate file permissions.
-func (c *Config) TLSCertFiles() (cert, key string, err error) {
-	if c.TLSCertFile == "" && c.TLSKeyFile == "" {
-		return "", "", nil
-	}
-
-	certFile, err := os.Stat(c.TLSCertFile)
```

Comment:
Is there a reason to remove these checks?

Reply:
They aren't really useful and don't guard against anything, the Go HTTP server will read these files before starting the HTTP server and return any errors.

https://cs.opensource.google/go/go/+/refs/tags/go1.21.0:src/net/http/server.go;l=3119-3122